src/rpc/net.cpp

```diff
@@ -156,7 +156,7 @@ static RPCHelpMan getpeerinfo()
                         {RPCResult::Type::NUM, "msg", "The total bytes received aggregated by message type\n"
                                                       "When a message type is not listed in this json object, the bytes received are 0.\n"
                                                       "Only known message types can appear as keys in the object and all bytes received\n"
-                                                      "of unknown message types are listed under '"+NET_MESSAGE_COMMAND_OTHER+"'."}
```

Comment:
Shouldn't this be in the previous commit?

Reply:
This change is done in src/rpc/net.cpp file. The first commit makes changes to src/net.cpp and src/net.h files.